core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.java

```diff
@@ -82,6 +77,10 @@ public KiwixWebView(Context context, WebViewCallback callback, AttributeSet attr
     settings.setUserAgentString(LanguageUtils.getCurrentLocale(context).toString());
     settings.setDomStorageEnabled(true);
     settings.setJavaScriptEnabled(true);
+    settings.setLoadWithOverviewMode(true);
+    settings.setUseWideViewPort(true);
+    setInitialScale(100);
```

Comment:
I don't think we need to set this

Reply:
I deleted `setInitialScale` and saw no difference, I am using gentoo wiki as the zim as it is the one cited in the ticket. What are you using?

Reply:
Try Vikidia

Reply:
Cool, good job